pom.xml

```diff
@@ -165,6 +170,14 @@ SOFTWARE.
           </execution>
         </executions>
       </plugin>
+      <plugin>
```

Comment:
@Sammers21 you don't need to configure compiler plugin by yourself, it's done in parent pom